### PR TITLE
Update/deploy/generate package.json files prior to installation

### PR DESCRIPTION
**Pull request by ricardo-valente**
Created pre-install script for generating the package.json file based on one of two possible templates (prd or dev), for each package. This script accepts 3 options:

- action: (-a | --action): rm or cp, default: cp
- env: (-e | --env): dev or prd, default: prd
- file: (-f | --file): default: package.json

### CODE REVIEW

File: scripts/dev/start.sh [hunk 1]
```diff
@@ -42,17 +42,21 @@ do
...
+echo "Running pre-installer..."
+yarn pre-install
+echo "packages ready for install \xF0\x9F\x9A\x80"
 
-if [ "$BUILD" = true ]
-then
-  echo "Running builder..."
-  docker-compose run build yarn build
-fi
+# if [ "$INSTALL" = true ]
```
Comment (rebecatalkdesk) [inline scripts/dev/start.sh:49]: Why did you comment this?

File: scripts/dev/start.sh [hunk 2]
```diff
@@ -42,17 +42,21 @@ do
...
+# then
+#   echo "Running installer..."
+#   docker-compose build ${SERVICE}
+# fi
 
-echo "Running watcher..."
-docker-compose run --service-ports --name ${NAME} ${SERVICE} concurrently "yarn docs" "yarn watch ${PACKAGE}"
\ No newline at end of file
+# if [ "$BUILD" = true ]
```
Comment (rebecatalkdesk) [inline scripts/dev/start.sh:55]: Why did you comment this?

**Review by rebecatalkdesk — APPROVED** [review]

File: scripts/dev/start.sh (hunk 2 above)
Comment (ricardo-valente) [reply scripts/dev/start.sh]: sorry my mistake

Comment (ricardo-valente) [reply scripts/dev/start.sh]: had to rebase my branch due to new changes done on package.json's in the meantime

**Pull request merged**